ledger/src/blocktree.rs

```diff
@@ -1847,6 +1847,34 @@ pub fn create_new_ledger(ledger_path: &Path, genesis_config: &GenesisConfig) ->
 
     blocktree.insert_shreds(shreds, None, false)?;
     blocktree.set_roots(&[0])?;
+    // Explicitly close the blocktree before we create the archived genesis file
+    drop(blocktree);
+
+    let archive_path = ledger_path.join("genesis.tar.bz2");
+    let mut args = vec!["jcfhS"];
+    args.push(archive_path.to_str().unwrap());
+    args.push("-C");
+    args.push(ledger_path.to_str().unwrap());
+    args.push("genesis.bin");
+    args.push("rocksdb");
```

Comment:
The mix of vec![] and push() seems a little odd to me.  How about just this: (with `cargo fmt`):
```rust
let mut args = vec!["jcfhS", archive_path.to_str().unwrap(),"-C",
   ledger_path.to_str().unwrap(), "genesis.bin", "rocksdb"];
```

Reply:
I think so too. Yay for cleaner code! eef79baf2086bb62f6a59b76a1c4ddfab61115f1